_docs/concepts/network-and-auth/auth.md

```diff
@@ -79,12 +80,13 @@ Istio Auth provides a per-cluster CA (Certificate Authority) to automate key and
 
 *   Revoke a specific key and certificate pair when necessary
 
-## Workflow
+For services running on VM/bare-metal machines, the above four operations are performed by Istio CA together with node agents.
 
-Istio Auth workflow consists of two phases, deployment and runtime. This section covers both of them.
+## Workflow
 
-### Deployment phase
+Istio Auth workflow consists of two phases, deployment and runtime. We discuss the first phase of the two scenarios separately since they are different. Once the key and certificate are deployed, the runtime phase is the same for the two scenarios. We briefly cover the workflow in this section and a more detailed version can be found [here](https://docs.google.com/document/d/1Grz9B0KS0vxqhdI-18iAfHk76cKLQnrFz-9njopRIEE/edit).
```

Comment:
The Istio Auth workflow consists of two phases:

This paragraph talks about scenarios. Which scenarios? In k8s vs VMs? The paragraph is really not very clear.

Reply:
I'm definitely not a fan of pointing folks to Google docs from this site. We should strive to be as self-contained as possible on this front.

Reply:
The two scenarios is k8s and VM, and it's mentioned in each subsection, e.g., "Deployment phase (Kubernetes Scenario)", I reworded it to make it more clear. 

I agree the gDoc is not perfect, but do you have any suggestions?

Reply:
The only suggestion I have would be to convert the doc into .md and add it to istio.io. This might not be desirable though either at this point...